src/Compilers/Core/Benchmarks/CreateSourceTextFromDisk.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+using Microsoft.CodeAnalysis.Text;
+
+#nullable disable
```

Comment:
Difficult to remove?

Reply:
There is so much `static` and instance state that

Reply:
It's one of those "it feels iffy" cases. Much of the state is `static` or instance where NRT doesn't flow that great. Went ahead and did the conversion though so we can see it. It's not too bad so I'm fine with keeping it.